micrometer-registry-newrelic-telemetry/runtime/src/main/java/io/quarkiverse/micrometer/registry/newrelic/telemetry/NewRelicConfig.java

```diff
@@ -0,0 +1,89 @@
+package io.quarkiverse.micrometer.registry.newrelic.telemetry;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.micrometer.runtime.config.MicrometerConfig;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+public class NewRelicConfig {
+
+    @ConfigRoot(name = "micrometer.export.newrelic", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+    public static class NewRelicBuildConfig implements MicrometerConfig.CapabilityEnabled {
+        /**
+         * Support for export to New Relic
+         * <p>
+         * Support for New Relic will be enabled if Micrometer
+         * support is enabled, the New Relic registry extension is enabled
+         * and either this value is true, or this value is unset and
+         * {@code quarkus.micrometer.registry-enabled-default} is true.
+         */
+        @ConfigItem
+        public Optional<Boolean> enabled;
+
+        @Override
+        public Optional<Boolean> getEnabled() {
+            return enabled;
+        }
+
+        /**
+         * By default, this extension will create a New Relic MeterRegistry instance.
+         * <p>
+         * Use this attribute to veto the creation of the default New Relic MeterRegistry.
+         */
+        @ConfigItem(defaultValue = "true")
+        public boolean defaultRegistry;
+
+        @Override
+        public String toString() {
+            return this.getClass().getSimpleName()
+                    + "{enabled=" + enabled
+                    + ",defaultRegistry=" + defaultRegistry
+                    + '}';
+        }
+    }
+
+    /**
+     * Runtime configuration for New Relic MeterRegistry
+     */
+    @ConfigRoot(name = "micrometer.export.newrelic", phase = ConfigPhase.RUN_TIME)
+    public static class NewRelicRuntimeConfig {
+        // @formatter:off
+        /**
+         * New Relic MeterRegistry configuration properties.
+         *
+         * A property source for configuration of the New Relic MeterRegistry to push
+         * metrics using the New Relic API, see https://micrometer.io/docs/registry/new-relic.
```

Comment:
Does New Relic have a guide for their registry that is separate from micrometer's? (I don't think so... )